### ISSUE TITLE
'System.Reflection.ReflectionTypeLoadException' when calling Assembly::GetTypes()

### ISSUE THREAD

**Issue by lextudio-support** [obfuscar/obfuscar#497]
Hi. I'm getting 'System.Reflection.ReflectionTypeLoadException'  when runnning this code:

Code:
```
            var dll = AppDomain.CurrentDomain.GetAssemblies().OrderBy(o => o.FullName);
            var types = dll
                   .SelectMany(o => o.GetTypes())
                   .Where(t => typeof(ISomeiface).IsAssignableFrom(t))
                   .Where(t => !t.IsAbstract)
                   .Where(t => !t.IsInterface)
            .Where(t => t.GetConstructor(new Type[] { }) != null);
```

**Original Reported Date**: 2016-05-28T04:20:59.897-07:00
**Planned For Release**: 
**Original CodePlex ID**: 35

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

Can you show how this code snippet triggers the exception and how this exception is related to Obfuscar?

**Original Posted Date**: 2016-06-10T21:16:57.49-07:00

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

Temp close.

**Original Posted Date**: 2016-09-21T05:48:28.797-07:00

**Comment by lextudio-support** [comment]
**Original Closed Date**: 2016-09-21T05:48:33.867-07:00
